docs/reference/cluster/update-settings.asciidoc

```diff
@@ -11,6 +11,14 @@ GET /_cluster/settings
 --------------------------------------------------
 // CONSOLE
 
+This API only returns settings explicitly defined. The default cluster settings can also be returned using the `?include_defaults` parameter:
```

Comment:
nit: maybe rephrase a bit to make clear only the above (parameterless) call returns only the explicitly defined settings. The API is capable of returning the defaults, as you are just about to explain now.